Add davidefpv rates

D4videAngeletti opened the pull request:
I wish to add my rates to the Betaflight community presets tab. Here is Davide FPV (Davide Angeletti), a Professional Italian Pilot and Vocational Trainer, Youtube content creator, and professional tuner. YT Channel:  [https://www.youtube.com/davidefpv](https://www.youtube.com/davidefpv) IG and TikTok: @davidefpv. 
Thanks a lot and happy flying!

limonspb commented on presets/4.4/tune/DavideFPV_DRock_3_Tune.txt:
```diff
@@ -0,0 +1,192 @@
+# -- Antigravity --
+
+set anti_gravity_gain = 115
+set craft_name = UNSTOPPABLE
```
plz remove craft_name

limonspb commented on presets/4.4/tune/DavideFPV_DRock_3_Tune.txt:
```diff
@@ -0,0 +1,192 @@
+set craft_name = UNSTOPPABLE
+set throttle_boost = 0
+
+#$ OPTION_GROUP BEGIN:PIDs
```
this option group does not have END

limonspb commented on index_hash.txt:
index_hash and index.json files should not be part of PR.
They are ok to test in your own separate branch.

limonspb commented on presets/4.4/rates/DavideFPVFreeStyle.txt:
```diff
@@ -0,0 +1,52 @@
+set yaw_srate = 59
+
+#$ OPTION BEGIN (UNCHECKED): Actual Rates for Slow FPV Freestyle
+set rates_type = ACTUAL
```
indentation

limonspb commented on presets/4.4/tune/DavideFPV_DRock_3_Tune.txt:
```diff
@@ -0,0 +1,192 @@
+
+    #$ OPTION END
+
+    #$ OPTION BEGIN (CHECKED): AgressiveFiltering approach (new/clean builds or open fiels flying | NO BMI270 Gyro)
```
Aggressive Filtering

limonspb commented on presets/4.4/tune/DavideFPV_DRock_3_Tune.txt:
```diff
@@ -0,0 +1,192 @@
+#$ TITLE: Davide FPV D-Rock 3 Tune
+#$ FIRMWARE_VERSION: 4.4
+#$ CATEGORY: TUNE
+#$ STATUS: OFFICIAL
```
#$ STATUS: EXPERIMENTAL

limonspb commented on presets/4.4/tune/DavideFPV_DRock_3_Tune.txt:
```diff
@@ -0,0 +1,192 @@
+    #$ OPTION BEGIN (CHECKED): 6S Lipo with HD Camera (1750-1850kv motors)
+
+	set iterm_relax_cutoff = 13
+	set simplified_master_multiplier = 125
```
in order for sliders to take effect you need to put
"simplified_tuning apply" at the end of a preset file
otherwise, this and all the following sliders commands will only move sliders, but not the actual values.

D4videAngeletti replied:
Thanks for reviewing. I'll correct everything and add this tune in another branch with relative pull request.

limonspb commented:
Plz don't skip my comment about index files.

Also might worth adding #$ discussion

More tips to follow:
https://github.com/betaflight/firmware-presets#tips-for-authors

limonspb commented:
> > Plz don't skip my comment about index files.
> > Also might worth adding #$ discussion
> > More tips to follow: https://github.com/betaflight/firmware-presets#tips-for-authors
> 
> Thanks Limon, I'm learning GIT Bash....sorry :-) Going to read your link and I'll add what you requested in a specific pull request for my tune preset Shall I add the "discussion" tag to this rates preset to complete the pull?

yes sure.
you can also disregard the recommendation of having 1 commit per PR. Because now we can squash when approving.
But all other recommendations are still live.

D4videAngeletti commented:
> 

Hi Lemon, I added what you requested. Please can you check the file and tell me if everything is ok? Thanks a lot!

limonspb commented:
> > 
> 
> Hi Lemon, I added what you requested. Please can you check the file and tell me if everything is ok? I put the preset in the 4.4 rates folder....I don't know if it's correct or if it's better to put it in the 4.3 rates one. Thanks a lot!

Hey! Idk why you keep ignoring my comment about index files, and the first tip in the tips for authors :)

D4videAngeletti commented:
> > > 
> > 
> > 
> > Hi Lemon, I added what you requested. Please can you check the file and tell me if everything is ok? I put the preset in the 4.4 rates folder....I don't know if it's correct or if it's better to put it in the 4.3 rates one. Thanks a lot!
> 
> Hey! Idk why you keep ignoring my comment about index files, and the first tip in the tips for authors link....

Hi Lemon, as you asked, I deleted index files and the Tune Preset (I didn't want to add that in this PR) with the commit [cc2cab3]. Am I wrong?

limonspb commented:
> > > > 
> > > 
> > > 
> > > Hi Lemon, I added what you requested. Please can you check the file and tell me if everything is ok? I put the preset in the 4.4 rates folder....I don't know if it's correct or if it's better to put it in the 4.3 rates one. Thanks a lot!
> > 
> > 
> > Hey! Idk why you keep ignoring my comment about index files, and the first tip in the tips for authors link....
> 
> Hi Lemon, as you asked, I deleted index files and the Tune Preset (I didn't want to add that in this PR) with the commit [[cc2cab3](https://github.com/betaflight/firmware-presets/commit/cc2cab3ac4c8bdcca2986bf1278888b9030ebaee)]. Am I wrong?

your PR should not change and should not remove index files.
it should leave them untouched. In the web view of the changed files you should see only your new preset

D4videAngeletti commented:
> 
> your PR should not change and should not remove index files. it should leave them untouched. In the web view of the changed files you should see only your new preset. You can also check other merged PRs for reference.

Hi Lemon! I'm sorry for bothering you again. I'm just learning how to use git and github (I learned a lot) but I can't find a good solution on how to remove "idex.json" and "index_hash" from this pull request :-( I googled around, I found how to rebase to a specific commit but my first commit in this PR had those file. Can you help me? Thanks a lot!!!

D4videAngeletti commented:
> > > your PR should not change and should not remove index files. it should leave them untouched. In the web view of the changed files you should see only your new preset. You can also check other merged PRs for reference.
> > 
> > 
> > Hi Lemon! I'm sorry for bothering you again. I'm just learning how to use git and github (I learned a lot) but I can't find a good solution on how to remove "idex.json" and "index_hash" from this pull request :-( I googled around, I found how to rebase to a specific commit but my first commit in this PR had those file. Can you help me? Thanks a lot!!!
> 
> copy your preset file somewhere. Then:
> 
> git checkout master git branch -D git checkout -b add-davidefpv-rates
> 
> copy back your preset file
> 
> git add -A git commit -m "Added davidefpv rates" git push origin add-davidefpv-rates -f

Done! How can I offer you a beer? :-)

limonspb commented:
> > > > your PR should not change and should not remove index files. it should leave them untouched. In the web view of the changed files you should see only your new preset. You can also check other merged PRs for reference.
> > > 
> > > 
> > > Hi Lemon! I'm sorry for bothering you again. I'm just learning how to use git and github (I learned a lot) but I can't find a good solution on how to remove "idex.json" and "index_hash" from this pull request :-( I googled around, I found how to rebase to a specific commit but my first commit in this PR had those file. Can you help me? Thanks a lot!!!
> > 
> > 
> > copy your preset file somewhere. Then:
> > git checkout master git branch -D git checkout -b add-davidefpv-rates
> > copy back your preset file
> > git add -A git commit -m "Added davidefpv rates" git push origin add-davidefpv-rates -f
> 
> Done! How can I offer you a beer? :-)

No problem. Any time you are in Texas :)
Like and subscribe LOL
https://www.youtube.com/channel/UCQtdpSBYlsWH6_m1Us_d8dg
